### DISCUSSION TITLE
iOS16 Beta

### DISCUSSION THREAD

**Discussion by uw206** [MiguelRipoll23/homebridge-securitysystem#258] [General]
Might be a premature discussion as it is still in beta, but all of the switches are showing the same system name. It's the same in Shortcuts as well, so I literally couldn't update or customize anything unless I use an iOS15 device. 

Perhaps it's an updated naming convention for the new iOS, or perhaps it's nothing. But I just thought you might want to be aware of this issue in case it persists when the official release becomes available.

**Comment by MiguelRipoll23** [comment]
Are these accessories from the plugin grouped or un grouped on the Home app? Thanks for the notice.

**Comment by uw206** [comment]
They're ungrouped.

**Reply by MiguelRipoll23** [reply]
Please see #285
